viz/charts/linkerd-viz/templates/metrics-api.yaml

```diff
@@ -71,9 +71,10 @@ spec:
       {{- include "linkerd.tolerations" (dict "Values" .Values.metricsAPI) | nindent 6 }}
       {{- end -}}
       {{- include "linkerd.node-selector" (dict "Values" .Values.metricsAPI) | nindent 6 }}
+      {{- include "linkerd.affinity" (dict "Values" .Values.metricsAPI) | nindent 6 }}
```

Comment:
Why was this line moved?

Reply:
Somehow the changes didn't work accordingly, and adding a line break at the end solved the problem

Reply:
I wan't able to reproduce the issue. Can you provide more details? (I don't understand the comment about empty component label).